build/relayers/ibc-relayers.md

```diff
@@ -16,6 +16,7 @@ We encourage everyone to follow and support our IBC-relayer operators
 | [Cosmonaut Stakes](https://twitter.com/CosmonautStakes)   | <p><img width="20px" src="https://user-images.githubusercontent.com/95667791/163506878-e7e278b2-1cf6-4dac-bc13-e92f63600ee0.png" alt="Twitter" data-size="line"> **Twitter:** <a href="https://twitter.com/CosmonautStakes">@CosmonautStakes</a><br><img width="20px" src="https://user-images.githubusercontent.com/95667791/163505650-fe08f2a7-8419-4a2f-97b7-e5182dd386e1.png" alt="Discord" data-size="line"> **Discord:** Danoly \| Cosmonaut Stakes#6003<br><img width="20px" src="https://user-images.githubusercontent.com/95667791/163506639-7d27e947-94c7-4740-bb46-8e5791e9fc6c.png" alt="Telegram" data-size="line"> **Telegram:** <a href="https://t.me/CosmonautStakes">@CosmonautStakes</a></p>                                  | <p>🔒 Delegate to **Cosmonaut Stakes** on: <a href="https://wallet.keplr.app/chains/persistence?modal=validator&chain=core-1&validator_address=persistencevaloper1rzauu3undh97yvdnj7wu2wwstm9wj8heeq2vcz&referral=true">Persistence</a></p>                                                                                                                                                                                                                |
+| [DSRV](http://dsrvlabs.com/) | <p><a href="mailto:validator@dsrvlabs.com">validator@dsrvlabs.com</a><br><img src="https://user-images.githubusercontent.com/95667791/163506878-e7e278b2-1cf6-4dac-bc13-e92f63600ee0.png" alt="Twitter" data-size="line"> Twitter <a href="http://twitter.com/dsrvlabs">@dsrvlabs</a><br><img src="https://user-images.githubusercontent.com/95667791/163506639-7d27e947-94c7-4740-bb46-8e5791e9fc6c.png" alt="Telegram" data-size="line"> Telegram @kim201212</p> | <p>🔒 Delegate to DSRV Validators on: <a href="https://wallet.keplr.app/#/osmosis/stake?modal=detail&#x26;chainId=osmosis-1&#x26;validator=osmovaloper1wlagucxdxvsmvj6330864x8q3vxz4x025rraa6">Osmosis</a>, Cosmos Hub, Presistence, Axelar, Agoric, Injective, Shetu, Crescent, Provenance, Tgrade, Terra, Stargaze, Chihuahua, Umee, Assetmantle |
```

Comment:
We're almost there @gnongs - if you can just include the same formatting settings for the icons as the other rows in the table, then we're good to go (now they look very big). Thanks!

Reply:
@dneorej I changed it !